app/scripts/menus.js

```diff
@@ -105,26 +106,37 @@ if (process.platform === 'darwin') {
 			label: 'View',
 			submenu: [
 				{
-					label: 'Reload current server',
-					accelerator: 'Command+R',
+					label: 'Original Zoom',
+					accelerator: 'Command+0',
 					click: function() {
-						const activeWebview = webview.getActive();
-						if (activeWebview) {
-							activeWebview.reload();
-						}
+						webFrame.setZoomLevel(0);
 					}
 				},
 				{
-					label: 'Reload application',
-					accelerator: 'Command+Shift+R',
+					label: 'Zoom In',
+					accelerator: 'Command+=',
```

Comment:
Why isn't this `<Ctrl>`+`<+>` to match what browsers use?

- `<Ctrl>`+`<+>` ... zoom in
- `<Ctrl>`+`<->` ... zoom out
- `<Ctrl>`+`<0>` ... reset zoom level

Reply:
It is already defined like that.